docs/en/String-Encryption.md

```diff
@@ -69,8 +69,11 @@ All encryption operations are included in `IStringEncryptionService`. You can in
 `IStringEncryptionService` methods has **passPharase** parameter with default value and it uses default PassPhrase when you don't pass passPhrase parameter. 
 
 ```csharp
-StringEncryptionService.Encrypt(value); // Default Pass Phrase
-StringEncryptionService.Encrypt(value, "MyCustomPassPhrase"); // Custom Pass Phrase
+// Default Pass Phrase
+StringEncryptionService.Encrypt(value);
```

Comment:
```suggestion
var encryptedValue = StringEncryptionService.Encrypt(value);
```

Assign return value to a dummy variable, so it will be much clear that it returns the encrypted value.